src/amo/components/AddonDescription.js

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+import { compose } from 'redux';
+
+import { nl2br, sanitizeHTML } from 'core/utils';
+import translate from 'core/i18n/translate';
+import Card from 'ui/components/Card';
+
+import './AddonDescription.scss';
+
+
+export const allowedDescriptionTags = [
+  'a',
+  'abbr',
+  'acronym',
+  'b',
+  'blockquote',
+  'br',
+  'code',
+  'em',
+  'i',
+  'li',
+  'ol',
+  'strong',
+  'ul',
+];
+
+export class AddonDescriptionBase extends React.Component {
```

Comment:
It looks like this component doesn't have anything specific to an add-on's description other than it using `addon.description`. You could use `children` instead and make this reusable.

```jsx
// Leaving `maxheight` out might be good in case we want to
// change it at different screen sizes.
<CollapsableCard maxheight={100} header={`About this ${addon.type}`}>
  <div dangerouslySetInnerHTML={prepareDescription(addon.description} />
</CollapsableCard>
```

It might be good to make the default be expanded and then collapse it in `componentDidMount()` if it is too tall. That way if JS is disabled you can still read the whole description.